dist/src/main/dist/conf/benchmark-report.py

```diff
@@ -1075,25 +1076,25 @@ def __init__(self):
 
     def addImage(self, plot):
         metric_name = plot.image_path.split('/')[-2]
-        
+
         encoded_image = ""
         with open(plot.image_path, "rb") as image_file:
-            encoded_image = base64.b64encode(image_file.read())    
+            encoded_image = base64.b64encode(image_file.read()).decode()
```

Comment:
```suggestion
            encoded_image = str(base64.b64encode(image_file.read()), encoding='utf-8')
```
IMO, the `decode` can be confusing here and `str` could make it clearer.

Reply:
@kwart your all reviews should be addressed now.